sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -1070,23 +1078,24 @@ def delayed_bundle_application(self,
     return beam_fn_api_pb2.DelayedBundleApplication(
         requested_time_delay=proto_deferred_watermark,
         application=self.construct_bundle_application(
-            op, current_watermark, element_and_restriction))
+            op.input_info, current_watermark, element_and_restriction))
 
   def bundle_application(self,
                          op,  # type: operations.DoOperation
                          primary  # type: SplitResultPrimary
                         ):
     # type: (...) -> beam_fn_api_pb2.BundleApplication
-    return self.construct_bundle_application(op, None, primary.primary_value)
+    assert op.input_info is not None
+    return self.construct_bundle_application(
+        op.input_info, None, primary.primary_value)
 
   def construct_bundle_application(self,
-                                   op,  # type: operations.DoOperation
+                                   op_input_info,  # type: operations.OpInputInfo
```

Comment:
Instead of passing around a `DoOperation` I switched this to a `OpInputInfo` because the latter is all that we need, and doing so let's us avoid having to assert that it is not `None` everywhere that it's used.

Reply:
Sounds good to me.